ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -314,6 +320,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                               blendDuration: 0)) {
             bottomOffset = 0
         }
+        UIAccessibility.post(notification: .layoutChanged, argument: self)
     }
 
     private func dismissAnimated() {
```

Comment:
don't you need to post on dismissanimated() too?

Reply:
> > > > did you do one finger swipe to right so that VO Focus moved from the "show" button to notificationview? how does swiftUI do "UIAccessibility.post(notification: .layoutChanged, argument: nil)" which what previous uikit notification do to give "double-beep" sound something has shown the in the screen and user should pay attention?
> > > 
> > > 
> > > I was able to get to the notification by swiping down with one finger.
> > 
> > 
> > right. so what we want is that VO automatically has the vo focus there when notification is shown.
> 
> Some conversation offline lead to this WWDC video from 2021, where at 26:00 they say SwiftUI doesn't have this and they just use UIAccessibility.post https://developer.apple.com/videos/play/wwdc2021/10119/

Scratch that, I just misunderstood them and @AccessibilityFocusState works.